doc/Language/faq.pod6

```diff
@@ -414,46 +380,33 @@ for @aa.kv -> $i, $_ { say "$i: $_" };     # OUTPUT: «0: 8␤1: 9␤2: 10␤3:
 say "$_.key(): $_.value()" for @aa.pairs;  # OUTPUT: «0: 8␤1: 9␤2: 10␤3: 11␤4: 12␤»
 =end code
 
-=head2 List: How can I concatenate lists
-
-Use the slip prefix (L<C<|>|/language/operators#prefix_|>) or the L<flat|/routine/flat> routine
-
-=begin code
-my @a = <a b c>;
-my @b = <d e f>;
-my @ab = |@a, |@b; say @ab;  # OUTPUT: «[a b c d e f]␤»
-say (@a, @b).flat;           # OUTPUT: «(a b c d e f)␤»
-=end code
-
-Note that flat recursively flattens L<non-containerized|/language/containers> L<iterables|/type/Iterable>.
-
-=head2 List: How can I generate a list of random numbers
+=head2 List: How can I generate a list of random numbers?
```

Comment:
Not a FAQ either. If it's not clear enough, or not searchable through that question, just add the key words to the description of "roll". In general, if the question is answered with "use x", just improve the description of X so that it can be found that way.